common-lib/src/main/scala/com/gu/mediaservice/lib/argo/ArgoHelpers.scala

```diff
@@ -24,6 +24,27 @@ trait ArgoHelpers extends Results {
     serializeAndWrap(response, Ok)
   }
 
+  def respondCollections[T](
+    uri: Option[URI] = None,
+    data: Map[String, Seq[T]],
+    links: List[Link] = Nil
+  ) (implicit writes: Writes[T]): Result = {
+    val collections = data.map({ case (key, collection) => {
+      key -> CollectionReponse(
```

Comment:
OK, just note that theseus will *not* understand these as embedded resources (or parse them as `Resource` objects) because they're not valid resources according to the spec. I guess it's just data that happens to look similar to a subresource and possibly helps future proofing (although could be slightly confusing).